src/Storage/Clients/IOndemandClient.cs

```diff
@@ -0,0 +1,18 @@
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Altinn.Platform.Storage.Clients
+{
+    /// <summary>
+    /// Interface for ondemand access
+    /// </summary>
+    public interface IOndemandClient
```

Comment:
Maybe OnDemand instead of Ondemant? 

```suggestion
    public interface IOnDemandClient
```

Reply:
Done